CHANGELOG.md

```diff
@@ -1,51 +1,57 @@
 # Changelog
 
+## 1.0.0
```

Comment:
Are we aiming for a GA release here? Just wondering as the previous version was a pre release version. Should it be 0.2.0?